### ISSUE TITLE
Make getRequestUrl accessible 

### ISSUE THREAD

**Issue by okigan** [signalfx/swagger-client-generator#9]
Often a REST interface provides a media object, like an image or video (and etc).

In such cases you actually need the url of the endpoint (rather than calling the endpoint by the swagger client).

So getRequestUrl (or something like that) should be public

**Comment by oztu** [comment]
I think this is reasonable, I'll look in to it, thanks for the suggestion.

**Comment by okigan** [comment]
bump

**Comment by oztu** [comment]
Alright just landed 0.2.13, you can use api.<resource>.<operationHandler>.getUrl(data) to figure out the url for a given resource. For example:

```
api.pet.getPetById.getUrl(2); // returns something like "http://pets.com/pet/2";
```
